tests/copyright_check/copyright_validator.py

```diff
@@ -0,0 +1,148 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+from __future__ import print_function
+
+import difflib
+
+import os
+import re
+import sys
+import pathlib
+import config
+
+print_logs = config.print_logs
+
+
+def fetch_samplefile():
+    """
+    Method to fetch sample copyright file
+    :return sample
+    """
+    sample = dict()
+    path = os.path.join(config.default_check_dir, config.default_sample_file)
+    ext = pathlib.Path(config.default_sample_file).suffix.split(".")[1]
+    sample_file = open(path, "r")
+    ref = sample_file.read().splitlines()
+    sample_file.close()
```

Comment:
Please use python context manager while opening files, this is important because if some exceptions would happen within the execution the file is guaranteed to be closed at the OS level and it is more pythonic

```python
    with path.open() as f:
        ref = f.read().splitlines()
```